js/base/Exchange.js

```diff
@@ -2277,6 +2278,17 @@ module.exports = class Exchange {
         return await this.createOrder (symbol, type, side, amount, price, query);
     }
 
+    async createStopOrder (symbol, type, side, amount, price = undefined, stopPrice = undefined, params = {}) {
+        if (!this.has['createStopOrder']) {
+            throw new NotSupported (this.id + 'createStopOrder() is not supported yet');
+        }
+        if (stop_price === undefined) {
```

Comment:
this check isn't necessary if we don't default `stopPrice` to be undefined, to do that we would need to put it before `price` and I don't like the idea of moving these parameters around, I think they should match the same order as `createOrder`